perfume-api/src/main/java/io/perfume/api/sample/application/port/SampleQueryRepository.java

```diff
@@ -0,0 +1,13 @@
+package io.perfume.api.sample.application.port;
+
+import io.perfume.api.sample.domain.Sample;
+
+import java.util.List;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'